Add SentiWS dataset for pos-tagging and sentiment-scoring (German) (#1504)
huggingface/datasets: datasets/senti_ws/README.md

---

> annotations_creators: []
> language_creators:
> - other

Reviewer: ```suggestion
annotations_creators:
- expert-generated
- machine-generated
language_creators:
- found
```

languages:
- de
licenses:
- cc-by-sa-3.0
multilinguality:
- monolingual
size_categories:
- n<1K

> source_datasets: []

Reviewer: ```suggestion
source_datasets:
- original
```

  - [Licensing Information](#licensing-information)
  - [Citation Information](#citation-information)

## Dataset Description

- **Homepage:** https://wortschatz.uni-leipzig.de/en/download
- **Repository:** [Needs More Information]
- **Paper:** http://www.lrec-conf.org/proceedings/lrec2010/pdf/490_Paper.pdf
- **Leaderboard:** [Needs More Information]
- **Point of Contact:** [Needs More Information]

### Dataset Summary

SentimentWortschatz, or SentiWS for short, is a publicly available German-language resource for sentiment analysis, opinion mining etc. It lists positive and negative polarity bearing words weighted within the interval of [-1; 1] plus their part of speech tag, and if applicable, their inflections. The current version of SentiWS contains around 1,650 positive and 1,800 negative words, which sum up to around 16,000 positive and 18,000 negative word forms incl. their inflections, respectively. It not only contains adjectives and adverbs explicitly expressing a sentiment, but also nouns and verbs implicitly containing one.

### Supported Tasks and Leaderboards

Sentiment-Scoring, Pos-Tagging

### Languages

German

## Dataset Structure

### Data Instances

> For pos-tagging:
> { 
> "word":"Abbau"
> "pos-tag": "NN"	
> }
> For sentiment-scoring:
> {
> "word":"Abbau"
> "sentiment-score":-0.058
> }

Reviewer: ````suggestion

For pos-tagging:

```
{ 
"word":"Abbau"
"pos_tag": 0
}
```

For sentiment-scoring:

```
{
"word":"Abbau"
"sentiment-score":-0.058
}
```
````

### Data Fields

SentiWS is UTF8-encoded text.

> For pos-tagging:
>                 {
>                     "word": datasets.Value("string"),
>                     "pos-tag": datasets.Value("string"),
>                 }
> For sentiment-scoring:
>                 {
>                     "word": datasets.Value("string"),
>                     "sentiment-score": datasets.Value("float32"),
>                 }

Reviewer: ```suggestion
For pos-tagging:
- word: one word as a string,
- pos_tag: the part-of-speech tag of the word as an integer,

For sentiment-scoring:
- word: one word as a string,
- sentiment-score: the sentiment score of the word as a float between -1 and 1,

```
